cypress/tests/integration/Plugins.spec.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file cypress/tests/integration/Plugins.spec.js
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ */
+
+ describe('Plugins tests', function() {
+    it('Check if Plugins.xml file was cached', function() {
+        cy.on('uncaught:exception', (err, runnable) => {
```

Comment:
It looks to me like this is used to avoid a failure do to an exception, but I'm not sure I see where that's necessary.

Reply:
Due to the XML processing on backend, sometimes this test ends abruptly with a "uncaught:exception" message, even when the cached file was created. :(

Reply:
Can you tell me a little more about this? Throwing away exceptions feels a little drastic!

Reply:
I've dropped this in favor of using `cy.intercept`